.eslintrc.json

```diff
@@ -0,0 +1,35 @@
+{
+  "parser": "babel-eslint",
+  "extends": "airbnb",
+  "rules": {
+    "semi": [2, "never"],
+    "prefer-arrow-callback": 0,
+    "space-before-function-paren": [2, "never"],
+    "func-names": 0,
+    "no-param-reassign": 0,
+    "comma-dangle": [2, "never"],
+    "prefer-rest-params": 0,
+    "max-len": [1, 150],
+    "no-console": 0,
+    "react/prefer-stateless-function": 0,
+    "react/prop-types": 0,
+    "react/jsx-no-bind": 0,
+    "react/jsx-closing-bracket-location": 0,
+    "react/no-deprecated": 0,
+    "react/jsx-boolean-value": 0,
+    "react/sort-comp": 0,
+    "prefer-template": 0,
+    "no-nested-ternary": 0,
+    "no-multi-spaces": 0
```

Comment:
We need it here https://github.com/atom-community/linter/blob/8b71dddc444297d1b0eb5af8b2280bc98c7e46d9/lib/linter-registry.js#L43